examples/notebooks/OllamaExperiment.ipynb

```diff
@@ -0,0 +1,383 @@
+      "File \u001b[0;32m~/prompttools/prompttools/experiment/experiments/ollama_experiment.py:99\u001b[0m, in \u001b[0;36mOllamaExperiment.ollama_completion_fn\u001b[0;34m(self, **params)\u001b[0m\n\u001b[1;32m     97\u001b[0m call_params \u001b[39m=\u001b[39m {k: v \u001b[39mfor\u001b[39;00m k, v \u001b[39min\u001b[39;00m params\u001b[39m.\u001b[39mitems() \u001b[39mif\u001b[39;00m k \u001b[39m!=\u001b[39m \u001b[39m\"\u001b[39m\u001b[39mclient\u001b[39m\u001b[39m\"\u001b[39m}\n\u001b[1;32m     98\u001b[0m \u001b[39mprint\u001b[39m(call_params)\n\u001b[0;32m---> 99\u001b[0m response \u001b[39m=\u001b[39m client(\u001b[39m*\u001b[39;49m\u001b[39m*\u001b[39;49mcall_params)\n\u001b[1;32m    100\u001b[0m \u001b[39mreturn\u001b[39;00m response\n",
+      "File \u001b[0;32m~/prompttools/prompttools/experiment/experiments/ollama/ollama_api.py:95\u001b[0m, in \u001b[0;36mOllamaAPI.generate\u001b[0;34m(self, model_name, prompt, logs)\u001b[0m\n\u001b[1;32m     93\u001b[0m data \u001b[39m=\u001b[39m {\u001b[39m\"\u001b[39m\u001b[39mmodel\u001b[39m\u001b[39m\"\u001b[39m: model_name, \u001b[39m\"\u001b[39m\u001b[39mprompt\u001b[39m\u001b[39m\"\u001b[39m: prompt}\n\u001b[1;32m     94\u001b[0m url \u001b[39m=\u001b[39m \u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39m{\u001b[39;00m\u001b[39mself\u001b[39m\u001b[39m.\u001b[39mbase_url\u001b[39m}\u001b[39;00m\u001b[39m/api/generate\u001b[39m\u001b[39m\"\u001b[39m\n\u001b[0;32m---> 95\u001b[0m response \u001b[39m=\u001b[39m requests\u001b[39m.\u001b[39;49mpost(url, json\u001b[39m=\u001b[39mdata,stream\u001b[39m=\u001b[39m\u001b[39mTrue\u001b[39;00m)\n\u001b[1;32m     96\u001b[0m response\u001b[39m.\u001b[39mraise_for_status()\n\u001b[1;32m     97\u001b[0m \u001b[39mif\u001b[39;00m response\u001b[39m.\u001b[39mstatus_code \u001b[39m==\u001b[39m \u001b[39m200\u001b[39m:\n\u001b[1;32m     98\u001b[0m     \u001b[39m# response.json() throws an error, which I think\u001b[39;00m\n",
+      "\u001b[0;31mAttributeError\u001b[0m: module 'requests' has no attribute 'post'"
+     ]
+    }
+   ],
+   "source": [
+    "experiment.run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "266c13eb",
+   "metadata": {},
+   "source": [
+    "## Evaluate the model response"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bebb8023",
+   "metadata": {},
+   "source": [
+    "To evaluate the results, we'll define an eval function. We can use semantic distance to check if the model's response is similar to our expected output."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "8ddbb951",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from prompttools.utils import similarity"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "e80dfeec",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "experiment.evaluate(\"similar_to_expected\", similarity.evaluate, expected=\"George Washington\")"
```

Comment:
We made some changes to how `.evaluate()` work. 

You likely want:
```python
from prompttools.utils import semantic_similarity
experiment.evaluate("similar_to_expected", semantic_similarity, expected=["George Washington"] * n)  # Replace `n` with the number of rows
```

See this [updated example](https://github.com/hegelai/prompttools/blob/main/examples/notebooks/OpenAIChatExperiment.ipynb).